client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useState, useCallback, useRef } from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Suggestions from '../../../../components/suggestions';
+import { STORE_KEY } from '../../store';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import { SiteVertical, isFilledFormValue } from '../../store/types';
+
+interface Props {
+	inputClass: string;
+}
+
+export default function VerticalSelect( { inputClass }: Props ) {
+	const popular = [
+		NO__( 'Travel Agency' ),
+		NO__( 'Digital Marketing' ),
+		NO__( 'Cameras & Photography' ),
+		NO__( 'Website Designer' ),
+		NO__( 'Restaurants' ),
+		NO__( 'Fashion Designer' ),
+		NO__( 'Real Estate Agent' ),
+	];
```

Comment:
Are these static in current signup? Do we know their id statically?

Reply:
I guess here's a comment that say these should match the ones returned by the Verticals API.
https://github.com/Automattic/wp-calypso/blob/master/client/components/site-verticals-suggestion-search/popular-topics.jsx

I was also thinking if there is a way to have ID's to match these or even better would be an extension of the Verticals API to return popular verticals for each type of site.

Reply:
Let's at least add a comment to explain why we keep them here rather than fetching from the REST API -- even if the explanation is just "They're not available from the API, and signup does the same thing".

Reply:
👋 @tyxla  - I believe you did some work with reusing verticals recently. Did you have to deal with these constants used in the WordPress.com signup flow?

Reply:
Hey @sirreal 😉 We were lucky to be able to reuse the existing component for selecting a site vertical (AKA site topic):

```
import SiteTopicForm from 'signup/steps/site-topic/form';
```

so we didn't have to directly work with fetching the verticals. I guess you folks could borrows some of the work in that component?